html-relative-links/index.html

```diff
@@ -10,12 +10,12 @@
 </head>
 <body>
   <ul>
-    <li>Home Page</li>
-    <li>About Me</li>
-    <li>Projects</li>
-    <li>Contact Us</li>
+    <li><a href="index.html">Home Page</a></li>
+    <li><a href="./about-me/about-me.html">About Me</a></li>
+    <li><a href="./projects/projects.html">Projects</a></li>
+    <li><a href="./contact-us/contact-us.html">Contact Us</a></li>
   </ul>
   <p>Hello Homepage!</p>
-  <img src="???" alt="Girl holding programming book">
+  <img src="./images/javascript.png" alt="Girl holding programming book">
```

Comment:
You don't need the `./`. Since we are at the root of our folder, and these folders are all siblings, you would just do the folder name directly. `about-me/about-me.html`